index.html

```diff
@@ -47,7 +46,7 @@
     - The freeCodeCamp Team
 
     -->
-    <!-- ========================================================================================================= -->
+    ======================================================================================================== -->
```

Comment:
try not  leaving  empty comment tags,because it will make your code not look clearn...

Reply:
- [ ] Nice job mentioning the reason for the changes you asked but Kindly ask for the changes in a direct and more polite way.